snapshot/src/main/scala/com/evolutiongaming/kafka/journal/BufferNr.scala

```diff
@@ -0,0 +1,37 @@
+package com.evolutiongaming.kafka.journal
+
+import com.evolutiongaming.scassandra.DecodeByIdx
+import com.evolutiongaming.scassandra.DecodeByName
+import com.evolutiongaming.scassandra.DecodeRow
+import com.evolutiongaming.scassandra.EncodeByIdx
+import com.evolutiongaming.scassandra.EncodeByName
+import com.evolutiongaming.scassandra.EncodeRow
+
+sealed abstract case class BufferNr(value: Int) {
+  override def toString: String = value.toString
+}
+
+object BufferNr {
+
+  def listOf(size: Int): List[BufferNr] =
+    (0 until size).toList.map(fromIntUnsafe)
+
+  def fromIntUnsafe(value: Int): BufferNr =
```

Comment:
1. some validation? 
2. non negative?
3. where is safe version? :)

Reply:
Fixed